# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/MathMLTextRunFactory.cpp
# Commit: 62f8ce89dce8
# Full Hash: 62f8ce89dce8f1162ff2f57debdd9a756a11a2f9
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-04-03 01:02:14
# Regressor Bug: 1619349
# File Overlap Count: 1
# Description:
#   Bug 1619349 - patch 1 - Record statistics about font-matching behavior in the presContext. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67787
# ==============================================================================

diff -r bcbee7191130 -r 62f8ce89dce8 layout/generic/MathMLTextRunFactory.cpp
--- a/layout/generic/MathMLTextRunFactory.cpp	Thu Apr 02 01:38:21 2020 +0000
+++ b/layout/generic/MathMLTextRunFactory.cpp	Thu Apr 02 01:38:18 2020 +0000
@@ -639,6 +639,7 @@
     params.explicitLanguage = styles[0]->mExplicitLanguage;
     params.userFontSet = pc->GetUserFontSet();
     params.textPerf = pc->GetTextPerfMetrics();
+    params.fontStats = pc->GetFontMatchingStats();
     params.featureValueLookup = pc->GetFontFeatureValuesLookup();
     RefPtr<nsFontMetrics> metrics =
         pc->DeviceContext()->GetMetricsFor(font, params);